src/main/java/com/github/manolo8/darkbot/core/objects/Gui.java

```diff
@@ -108,8 +109,8 @@ public void moveWindow(int newX, int newY) {
 
     public boolean show(boolean value) {
         if (trySetShowing(value)) {
-            if (minimizable) toggleVisibility();
-            else legacyToggle(value);
+            if (!(minimizable && toggleVisibility()) && !(closable && close()))
+                legacyToggle(value);
```

Comment:
```suggestion
            if (minimizable ? !toggleVisibility() : closeable ? !close() : false) {
               legacyToggle(value);
             }
```

This logic is kinda confusing and hard to follow, what exactly is the intended flow?

if minimizable -> toggleVisibility (and fallback to legacyToggle)
if closeable -> close (and fallback to legacyToggle)
if neither? do nothing? or legacyToggle?

Shouldn't `value` be used within this method? (ie, what the end goal state is) such that `visible ? toggleVisibility() : close()` ?